bumpCversion.py

```diff
@@ -110,21 +117,50 @@ def get_target_files(args):
 
     return target_files
 
+def valid_version_congruence(args, target_files, versions):
+    """
+    Check to see if version number is the same across all files.
+    """
+    while target_files:
+        filetype = get_filetype_object(args, target_files)
+        # add version to unique set
+        versions.add(filetype.version_number.__str__())
```

Comment:
I think it would be better to use the string method instead of the "special" string method of our class.
`versions.add(str(filetype.version_number))`